plugins/producer_plugin/include/eosio/producer_plugin/subjective_billing.hpp

```diff
@@ -114,6 +114,7 @@ class subjective_billing {
 public:
    void disable() { _disabled = true; }
    void disable_account( chain::account_name a ) { _disabled_accounts.emplace( a ); }
+   bool is_account_disabled(const account_name & a ) { _disabled_accounts.count( a ); }
```

Comment:
Prefer: `const account_name& a` vs` const account_name & a`.
Add `return`.

Reply:
Done. Addressed in 60b4a62f0. Good catch.